### PR TITLE
[Helix] Go back to 1604 for now

### PR DESCRIPTION
**Pull request by HaoK**
1804 doesn't have enough cores to support our volume right now

### CODE REVIEW

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true'">
     <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
```
Comment (BrennanConroy) [inline eng/targets/Helix.Common.props:32]: ?

Comment (HaoK) [reply eng/targets/Helix.Common.props:32]: This makes it so we run 1604 on PRs, and only use 1804 on the 2x a day master builds, this is what we were doing before I believe

Comment (HaoK) [reply eng/targets/Helix.Common.props:32]: so 1804 wouldn't be used at all on internal builds as those only run required checks now.  helix matrix doesn't run on internal

**Review by BrennanConroy — APPROVED** [review]

File: eng/targets/Helix.Common.props (hunk 1 above)
Comment (dougbu) [reply eng/targets/Helix.Common.props:32]: Why change anything in the public runs? The only issue is internal builds.

I'd prefer making this conditional on the `$(SYSTEM_TEAMPROJECT)` value. Perhaps even don't bother testing on Ubuntu in official builds.

**Review by dougbu — CHANGES_REQUESTED** [review]

**Comment by HaoK** [comment]
1604 has been very reliable for months, 1804 already ran out of disk space issues yesterday, and is overloaded interally today.  It doesn't seem worth switching from something that's worked well for a long time, to get additional coverage of 4 CDN tests that are still running on all other queues

**Review by dougbu — APPROVED** [review]
Discussed more offline&hellip;

**Review by halter73 — APPROVED** [review]

**Pull request merged**